gazebo/physics/simbody/SimbodyLink.cc

```diff
@@ -505,44 +513,69 @@ void SimbodyLink::AddForce(const ignition::math::Vector3d &_force)
 }
 
 /////////////////////////////////////////////////
-void SimbodyLink::AddRelativeForce(const ignition::math::Vector3d &/*_force*/)
+void SimbodyLink::AddRelativeForce(const ignition::math::Vector3d &_force)
 {
-  gzerr << "Not implemented.\n";
+  auto const worldForce = this->WorldPose().Rot().RotateVector(_force);
+  this->AddTorque(worldForce);
```

Comment:
I'm confused why we would call `AddTorque` from the `AddRelativeForce` method

Reply:
I'm sorry, this is a mistake I made. Line 519 should be a call to this->AddForce. I fixed this in the latest commit.